src/TourManager/TourManager.Common/Clients/Activity.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TourManager.Common.Clients
+{
+    public class Activity
+    {
+        public long? Id { get; set; }
+        public ActivityType Type { get; set; }
+        public string Name { get; set; }
```

Comment:
Please consider to use non-forgiving operator which can't be null.

Reply:
for the first time (insert new activity it can be null)

Reply:
Please comment that in the code as a description.